03_analysis/compare_ffs.py

```diff
@@ -646,7 +726,7 @@ def main(in_dict, read_pickle, conf_id_tag, plot=False, mol_slice=None):
 
     Parameter
     ---------
-    in_dict : OrderedDict
+    in_dict : Orderedict
```

Comment:
The D in [OrderedDict](https://docs.python.org/3/library/collections.html#collections.OrderedDict) should be capitalized

Reply:
Resolved in f0d5033